compiler/env/OMRCPU.hpp

```diff
@@ -131,6 +131,8 @@ class CPU
    bool isI386() { return _minorArch == TR::m_arch_i386; }
    bool isAMD64() { return _minorArch == TR::m_arch_amd64; }
 
```

Comment:
This API should have the same Doxygen comment as the platform-specific ones.   It is the API used by architectures that don't have TM support yet (e.g., ARM32, AArch64).

Reply:
Good point, missed that

Reply:
Updated (see https://github.com/eclipse/omr/compare/9142f82165c1480c850be7146abc18c6ecc4fd46..0193bf0dfe329515a899e6ef22c35ef6fcae37ec)